### DISCUSSION TITLE
How do I setup an autocommand for Nvdash?

### DISCUSSION THREAD

**Discussion by amagow-ct** [NvChad/NvChad#1924] [Q&A] [answered]
I am not very good at configuring neovim, I would like to run the `Nvdash` command when I open nvim. I know I could use and autocommand for this, and the event to use would be `VimEnter` but don't know what is the pattern in this case. As I would only like to open the dash when I am opening a directory. Not when I am opening a file.

Thanks 🙏

**Comment by siduck** [comment, marked as answer]
hi @amagow-ct , just enable load_on-startup option in chadrc. 

```lua
M.ui = {
  theme = "everforest",

  nvdash = {
    load_on_startup = true
  }
}
```
